java/org/contikios/cooja/radiomediums/UDGM.java

```diff
@@ -188,8 +188,6 @@ protected RadioConnection createConnections(Radio sender) {
            it will be activated later when the radio will be
```

Comment:
I don't know enough about the radio mediums to comment about the functional change in this PR, but does this comment concern the code that is being removed?